sdks/java/io/common/src/test/java/org/apache/beam/sdk/io/common/IOTestPipelineOptions.java

```diff
@@ -107,4 +107,11 @@
   String getCompressionType();
 
   void setCompressionType(String compressionType);
+
+  /* Xml */
```

Comment:
<!--thread_id:cc_171360003_t; commit:e3960b9fea070f4d514b71629fad156fc4f8773c; resolved:0-->
<!--section:context-quote-->
> **lgajowy** wrote:
> Ok

<!--section:body-->
Seems like this hasn't been done yet.